v-next/hardhat-utils/src/ci.ts

```diff
@@ -0,0 +1,33 @@
+/**
+ * Checks whether the current process is running in a CI environment.
+ *
+ * @returns True if the current process is running in a CI environment.
+ */
+export function isCi() {
+  const env = process.env;
+
+  return !!(
+    isGithubActions() ||
+    isNow() ||
+    isAwsCodeBuild() ||
+    env.CI !== undefined || // Travis CI, CircleCI, Cirrus CI, GitLab CI, Appveyor, CodeShip, dsari
+    env.CONTINUOUS_INTEGRATION !== undefined || // Travis CI, Cirrus CI
+    env.BUILD_NUMBER !== undefined || // Jenkins, TeamCity
+    env.RUN_ID !== undefined || // TaskCluster, dsari
+    false
```

Comment:
`|| false` doesn't add anything, does it?

Reply:
[Done in this commit](https://github.com/NomicFoundation/hardhat/pull/5340/commits/d5ce13f963d5f7450bb946cb3c0933e48f895730)